tests/e2e/taqueria-plugin-flextesa.spec.ts

```diff
@@ -0,0 +1,242 @@
+    //                 "ken": {
+    //                     "initialBalance": "1"
+    //                 },
+    //                 "tran": {
+    //                     "initialBalance": "2"
+    //                 },
+    //                 "mathew": {
+    //                     "initialBalance": "3"
+    //                 }
+    //             },
+    test('Verify that taqueria flextesa plugin can start and stop a sandbox with custom name', async () => {
+        try {
+            // Setting up docker container name
+            dockerName = "test"
+
+            // 1. Updating config to add test sandbox
+            execSync(`cp e2e/data/config-flextesa-test-sandbox.json ${taqueriaProjectPath}/.taq/config.json`);
+
+            // 2. Run sandbox start command with customer name
+            const stdoutStart = execSync(`taq start sandbox ${dockerName}`, {cwd: `./${taqueriaProjectPath}`}).toString().trim();
+
+            // 3. Verify that sandbox has been started and taqueria returns proper message into console
+            expect(stdoutStart).toEqual("Started test.");
+
+            // 4. Verify that docker container has been started
+            const [,dockerContainerTest] = execSync(`docker ps --filter "name=${dockerName}"`).toString().trim().split(/\r?\n/);
+            expect(dockerContainerTest).toBeDefined();
+
+            // 5. Run list accounts command and verify that it returns list of default accounts
+            await waitForExpect(() => {
+                const stdoutList = execSync(`taq list accounts ${dockerName}`, {cwd: `./${taqueriaProjectPath}`}).toString().trim();
+                expect(stdoutList).toContain("bob");
+                expect(stdoutList).not.toContain("jane");
+            });
+
+            // 6.  Run stop command and verify the output
+            const stdoutStop = execSync(`taq stop sandbox ${dockerName}`, {cwd: `./${taqueriaProjectPath}`}).toString().trim();
+
+            // 7. Verify that taqueria returns proper message into console
+            expect(stdoutStop).toEqual("Stopped test.");
+
+        } catch(error) {
+            throw new Error (`error: ${error}`);
+        }
+
+    });
+
+
+    // TODO: Need to clarify about updating accounts, otherwise there is no way to do step d.
+    test.skip('Verify that taqueria flextesa plugin can retrieve data from updated config after restart', async () => {
+        try {
+            // Setting up docker container name
+            dockerName = "local"
+            //      a. Start sandbox
+            //      b. Check balance
+            //      c. Stop sandbox
+            //      d. Update config
+            //      e. start sandbox again
+            //      f. Check balance again and see that it changed
+
+
+        } catch(error) {
+            throw new Error (`error: ${error}`);
+        }
+
+    });
+
+
+    // Clean up process to stop container if it was not stopped properly during the test
+    afterEach( () => {
```

Comment:
Yeah like I said above I think it is better to have tests clean up after themselves rather than have an afterEach clause that will just try to find something and then try to stop it

Reply:
It looks by container name - not just everything - `docker ps --filter "name=${dockerName}"`
It is smart enough to filter out only what we need.